services/director/src/simcore_service_director/producer.py

```diff
@@ -253,6 +292,20 @@ async def _create_docker_service_params(
             docker_params["task_template"]["Placement"]["Constraints"] += param["value"]
         elif param["type"] == "Constraints":  # REST-API compatible
             docker_params["task_template"]["Placement"]["Constraints"] += param["value"]
+        elif param["name"] == "env":
+            log.debug("Found env parameter %s", param["value"])
+            env_settings = _parse_env_settings(param["value"])
+            if env_settings:
+                docker_params["task_template"]["ContainerSpec"]["Env"].update(
+                    env_settings
+                )
+        elif param["name"] == "mount":
+            log.debug("Found mount parameter %s", param["value"])
+            mount_settings = _parse_mount_settings(param["value"])
+            if mount_settings:
+                docker_params["task_template"]["ContainerSpec"]["Mounts"].append(
+                    mount_settings
+                )
```

Comment:
minor: in the last iteration here I was trying to be 100% compatible with how the REST-API defines these parameters such that it would be possible to just read the value directly from the docker label without having to parse anything. (that is the idea of what you have on line 294 for example)

Reply:
Good point. So you suggest to leave the responsibility for not making mistakes to the creator of the label?

Reply:
no, we can still double-check. what I meant is that the label would look exactly the same as how the REST-API defines it so we do not have to convert stuff. as aiodocker takes REST-API like parameters. but that is minor.